packages/jest-matcher-utils/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "jest-matcher-utils",
   "description": "A set of utility functions for jest-matchers and related packages",
-  "version": "17.0.1",
+  "version": "17.0.2",
```

Comment:
we do this automatically when we publish new versions. so no need to update it here :)